site/installation/README.md

```diff
@@ -96,4 +96,4 @@ kpt help
 [cloud-sdk]: https://github.com/GoogleCloudPlatform/cloud-sdk-docker
 [linux]: https://github.com/GoogleContainerTools/kpt/releases/download/v1.0.0-alpha.4/kpt_linux_amd64
 [darwin]: https://github.com/GoogleContainerTools/kpt/releases/download/v1.0.0-alpha.4/kpt_darwin_amd64
-[migration guide]: installation/migration/
+[migration guide]: installation/migration
```

Comment:
so both https://kpt.dev/installation/migration/ and https://kpt.dev/installation/migration works after the change ? I think this needs a broader fix(as a separate PR) as the urls don't work without trailing /

Reply:
Unfortunately, just the latter link (https://kpt.dev/installation/migration) will work.
I agree: this is tracked in a separate issue but is a problem with Docsify's history mode. There are some workaround we can explore but the best fix would be at the Docsify level.

Reply:
Speaking of slashes, the rest of the docs have a leading slash:

/installation/migration

Also, don't we have a CI linter that catches broken links? If not, it should be added soon.

Reply:
We do; the link just isn't broken.